e2e/integration/searchtests.js

```diff
@@ -81,14 +78,17 @@ context( 'Test search', () => {
         cy.get( '#select-checkbox-expanded-toggle' ).first().click( { force: true } );
     } )
 
+    it( 'Test for sort application', () => {
+        cy.get('#select-checkbox-expanded-toggle')
+    })
     it( 'Test for invalid search', () => {
-        cy.wait( 1500 );
-        cy.get( 'opc-nav-search[slot="opc-nav-search"]' ).within( () => {
-            cy.get( '.opc-nav-search__btn' ).click( { force: true } );
-            cy.wait( 500 );
-            cy.get( 'input[name="query"]' ).click( { force: true } ).clear( { force: true } ).type( 'qwertyuiopasdgthchgbtnjkmnvxs', { force: true } );
-            cy.get( '.opc-nav-search__btn' ).click( { force: true } );
-        } )
+        cy.wait( 15000 );
+        cy.get('.opc-nav-search__btn', { timeout: 5000 } ).click( { force: true } );
+        cy.wait(3000)
+        cy.get( 'input[name="query"]',{timeout:5000} ).focus().type( 'qwertyuiopasdgthchgbtnjkmnvxs', { force: true } );
```

Comment:
Is there a specific reason to give a random string @mudit94 ?
I guess you can add something which is relevant.

Reply:
Hi @hybridx , random string is given to check for invalid search response